content/presubmission_inquiry.md

```diff
@@ -42,7 +42,7 @@ These rules are chosen so that readers of varying experience levels will be able
 
 Upon determination that this manuscript will be suitable for submission to PLOS Computational Biology, I will further engage with the community to solicit feedback and contributions. All authors of the paper will meet the ICMJE authorship standards.
 
-It is my hope that, by providing guidance on deep learning, these powerful methods can be properly utilized by the community and increase the number of biological researchers using DL (by making it more inviting) and the quality of the research (by helping them avoid common mistakes), ultimately leading to a deeper understanding of biology.
+By providing guidance on deep learning, powerful DL methods can be properly utilized by the community towards a deeper understanding of biology. Here, we aim to increase the number of biological researchers that take advanatage of DL methods (by making them more inviting) and improve the overall the quality and reproducibility of the research (by helping them avoid common mistakes).
```

Comment:
One minor thing: the use of "them" in the last sentence might be confusing since the first use refers to the deep learning methods and the second refers to the researchers. 

Once you clarify, it's ready to merge.

Reply:
Thanks for the help!